bluemira/builders/EUDEMO/reactor.py

```diff
@@ -63,7 +64,6 @@ def run(self) -> Component:
         component.add_child(self.build_plasma())
         component.add_child(self.build_TF_coils(component))
         component.add_child(self.build_PF_coils(component))
-        component.add_child(self.build_first_wall(component))
```

Comment:
@CoronelBuendia I pictured calling `build_in_vessel_component_shapes()` here, then using those shapes to call whichever other builders need them. If you had something else in mind, feel free to go with that.

The `build_ivc_xz_shapes()` function will turn the output components of the `IVCBuilder` into the shapes you need